website/__init__.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env/ python
+
+from flask import Flask, session
+from .interfacing import Interfacer
+from .managers import LoginManager, UserManager, GameManager, TagManager
+from .generateFakeData import generateData
+
+
+config = { 
+        'database' : 'ZoomFront' 
+}
```

Comment:
This config option can do nothing but break the application, as no database except `ZoomFront` can currently be generated. 

Also, Flask provides a built in way to manage config, you can merge your config with the `app.config` dict through a few of these loading strategies: https://flask.palletsprojects.com/en/3.0.x/config/#configuring-from-python-files. This should all happen wherever the app factory ends up, so most likely in a top level `create_app.py`. See the comments further down this file.